sphinx_gallery/gen_rst.py

```diff
@@ -93,6 +97,18 @@ def flush(self):
         self.file2.flush()
 
 
+class MyBytesIO(BytesIO):
```

Comment:
I couldn't think of a better one either, but if someone else has an idea I'm happy to change it. I used the `My` prefix because eventually it goes to a variable named `my_buffer`.